package.json

```
@@ -67,7 +71,7 @@
     "validate-commit-msg": "^2.14.0"
   },
   "peerDependencies": {
-    "eslint": "^7 || ^8.2.0"
+    "eslint": "^8.2.0 || ^9.0.0"
```

```suggestion
  },
  "peerDependencies": {
    "eslint": "^8.57.1 || ^9.14.0"
```
might as well not bother supporting anything that's already out of date.